apps/web/src/components/base-org/shared/TopNavigation/index.tsx

```diff
@@ -92,6 +93,12 @@ const links: TopNavigationLink[] = [
 ];
 
 export default function TopNavigation() {
+  const pathname = usePathname();
+  const showGasDropdownAndConnectWallet =
```

Comment:
Could we write this as something like `!['/jobs', '/about', etc.].includes(pathname)`?

Reply:
yep! updated now